guides/release/components/template-lifecycle-dom-and-modifiers.md

```diff
@@ -381,14 +385,14 @@ export default class AudioPlayerComponent extends Component {
 }
 ```
 
-Now the component is managing the state of the player, but we still need to translate that into the DOM method calls. So let's create a modifier!
+That's it for the component: we're translating the user's interactions into *state*. Now we need to build a modifier to translate the state into the appropriate DOM method calls!
```

Comment:
I like these sentences. They reminded me of composing functions in math.

Reply:
Don’t tell anyone, but I’m kind of obsessed with composition in the mathematical sense as a key ingredient of software design. 🤓